### ISSUE TITLE
I don’t understand what I’m doing wrong with the template

### ISSUE THREAD

**Issue by hey-com** [richtabor/wabi#18]
Hello
I make a website for my son
I use the Wabi theme
<a href="https://richtabor.com/wabi/" rel="noopener" target="_blank">https://richtabor.com/wabi/</a>
The whole site, according to the old habit, is made of pages.
sample on the website of the theme creator here
<a href="https://richtabor.com/about/" rel="noopener" target="_blank">https://richtabor.com/about/</a>
Unfortunately, due to the fact that Russian words are quite long (for example, the word “Pharmaceutical” placed in the title does not fit into the line when viewed from a smartphone), I use the No Title layout.
The title is now the first line on the site page.
Setting up this page was quite a hassle, and now for the new page I duplicate the main page and then redo it.
It's long.
I read that can create page templates yourself and made it.
I made a <a href="http://prntscr.com/bQEOSyvLyw8Q" rel="noopener" target="_blank">test page</a> and applied the template I made to it.
What was I waiting for?
I was expecting that the page, when applying the <a href="http://prntscr.com/r_npV-vyNQrc" rel="noopener" target="_blank">new template</a>, would be transformed in the administrative part, and I would be able to remake it to the one I need, fill it with text.
That did not happen.
Then I looked it up on the net - and hooray, <a href="https://kugno.ru/%d1%8d%d1%82%d0%be-%d1%82%d0%b5%d1%81%d1%82%d0%be%d0%b2%d0%b0%d1%8f-%d1%81%d1%82%d1%80%d0%b0%d0%bd%d0%b8%d1%86%d0%b0/" rel="noopener" target="_blank">it began to look right!</a>
I clicked on the Edit page button - I was thrown into the administrative part - and I see the test page again.
I tried everything yesterday, but there is no other way - I see my template on the web, it is linked to the page - but I cannot edit it.
Please tell me what I did not take into account and what I am doing wrong?
Thanks

**Comment by jenxi** [comment]
I don’t think this is an issue with the theme. It is easy to test. Switch to the default theme and see if this problem still persists.

**Comment by richtabor** [comment]
I’m not quite following. The page title will show within the editor, regardless of the applied template. But if you use the template that removes the page title, you should not see a title on the front end.
